docs/2.drivers/netlify.md

```diff
@@ -25,7 +21,25 @@ const storage = createStorage({
 });
 ```
 
-You can create a deploy-scoped store by settings `deployScoped` option to `true`. This will mean that the deploy only has access to its own store. The store is managed alongside the deploy, with the same deploy previews, deletes, and rollbacks.
+You can use Unstorage in a Netlify edge function without installing any dependencies:
+
+```js
+import { createStorage } from "https://esm.sh/unstorage";
```

Comment:
I would very love to advocate URL import patterns but it would be inconsistent to only do it here and also it probably deserves more explanation docs that for example Node.js/bundlers don’t support it out of the box at least.

If you are happy shall we defer adding this section to another PR?

Reply:
This example is specific to Netlify Edge Functions though. They need this syntax if they're not being bundled by a framework because subpath imports from node modules don't work. I can see if I can find some wording that's clearer.

Reply:
A more detailed explanation would be cool! BTW for Nuxt/Nitro users we do support it without CDN (only)

Reply:
Yeah, it would work with anything that bundles it. It's just when doing a raw import, and is a limit of our current npm handling. It even works fine in this project when I manually import the compiled file. How's the updated wording?